core/src/integration-test/java/com/scalar/db/util/CosmosAdminTestUtils.java

```diff
@@ -0,0 +1,78 @@
+package com.scalar.db.util;
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosException;
+import com.azure.cosmos.CosmosStoredProcedure;
+import com.azure.cosmos.models.CosmosItemRequestOptions;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.util.CosmosPagedIterable;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.storage.cosmos.CosmosAdmin;
+import com.scalar.db.storage.cosmos.CosmosConfig;
+import com.scalar.db.storage.cosmos.Record;
+import java.util.Properties;
+
+public class CosmosAdminTestUtils extends AdminTestUtils {
```

Comment:
Is it better that these `XXXAdminTestUtils` classes in the storage specific packages (`com.scalar.db.storage.xxx`)?

Reply:
Yes it would be but I don't think it is possible with the current integration-testing package structure since the factory like `AdminTestUtils.create()` method reference all storage implementation classes.

Let's say each `XXXAdminTestUtils` classes are moved to their respective `com.scalar.db.storage.xxx` storage package. Because of the `AdminTestUtils.create` method, all storage implementations are imported in `AdminTestUtils`. Because of this, when running the integration test in `com.scalar.db.storage.jdbc` that are using `com.scalar.db.util.AdminTestUtils`, it requires `com.scalar.db.storage.dynamo.DynamoAdminTestUtils` and others storage implementation to be present in the classpath, which is not the case because storage test package are compartmentalized from each other. 
If possible, I would like to keep the `AdminTestUtils.create` method as it simplify the instantiation. 
What do you think ?

Reply:
I understand. Let's keep it as it is. Thanks.